frontends/jest-shared-setup.ts

```diff
@@ -59,32 +58,17 @@ const polyfillResizeObserver = () => {
 }
 polyfillResizeObserver()
 
-failOnConsole()
-
-configure({
-  /**
-   * Adapted from https://github.com/testing-library/dom-testing-library/issues/773
-   * to make the error messages a bit more succinct.
-   *
-   * By default, testing-library prints much too much of the DOM.
-   *
-   * This does change the stacktrace a bit: The line causing the error is still
-   * there, but the line where the error is generated (below) is most visible.
-   */
-  getElementError(message, _container) {
-    const error = new Error(message ?? "")
-    error.name = "TestingLibraryElementError"
-    return error
-  },
-})
```

Comment:
The comment here said:
> By default, testing-library prints much too much of the DOM.

This doesn't seem to change how much of the DOM gets printed, and the DOM elements it does print are helpful anyway, IMO. (I guess my opinion has changed since 22 months ago.)